src/yajra/Zillow/ZillowClient.php

```diff
@@ -48,9 +48,9 @@ class ZillowClient
     protected $results;
 
     /**
-     * @var array
+     * @var Illuminate\Support\Collection
```

Comment:
I think we need to include `illuminate\support` on composer if we are going to use collection?

Reply:
Good call :) I've pushed a new commit to add it

Reply:
I had added it in but when I was testing it resulted in some issues - I'll try to test more later as well as test getPhotos so you can merge